src/routes/klageskjema/begrunnelse/begrunnelse.tsx

```diff
@@ -84,7 +87,10 @@ const Begrunnelse = ({ klage }: Props) => {
 
     useEffect(() => {
         if (klage.vedtakType === vedtakType && klage.vedtakDate === vedtakDate && klage.fritekst === fritekst) {
+            setAutosaveStatus(AutosaveStatus.SAVED);
             return;
+        } else {
```

Comment:
`else` er unødvendig, ´return´ på linjen over vil sørge for at koden under aldri blir kjørt.